### PR TITLE
bonus content alignment fix

### PR DESCRIPTION
**Pull request by smitsanghvi21**

### CODE REVIEW

File: site/templates/tv/detail.jet [hunk 1]
```diff
@@ -81,13 +81,34 @@
...
+          
           {{if len(tvseason.Bonuses) > 0 }}
-            <div class="meta-detail-bonus-content d-flex">
+            <div class="meta-detail-bonus-content"">
               <h4>{{i18n("meta_detail_bonus_title")}}</h4>
-              {{yield slider(itemsPerRow=4, itemLayout="landscape") content}}
+              {{yield list(itemsPerRow=4, itemLayout="landscape") content}}
                 {{range bonus := tvseason.Bonuses}}
                   <div class="bonus-item">
-                    {{yield bonusItem(item=bonus)}}
```
Comment (stajs) [inline site/templates/tv/detail.jet:90]: Why did you change from `yield`ing to have it inline?

File: site/templates/tv/detail.jet [hunk 2]
```diff
@@ -97,6 +118,10 @@
       </div>
         </div>
       </div>
+    </div>
+      </div>
+        </div>
+      </div>
```
Comment (stajs) [inline site/templates/tv/detail.jet:124]: What's happened to this alignment? Looks a bit odd.

**Review by stajs — CHANGES_REQUESTED** [review]
For the SASS file, it looks like it's auto-formatted a lot of lines which makes it too hard to see what the change is. Can you please highlight the actual change there?

File: site/styles/_meta-detail.scss [hunk 3]
```diff
@@ -346,6 +350,9 @@
   padding: 0;
   // border-top: 1px solid rgba(255,255,255,0.2);
   padding-top: 20px;
+  .poster .s72-btn-play {
+    padding-right: 3px;
+  }
```
Comment (smitsanghvi21) [inline site/styles/_meta-detail.scss:355]: This was added to the scss file.

File: site/styles/_meta-detail.scss [hunk 4]
```diff
@@ -560,9 +568,15 @@
...
     }
   }
 }
+
+.meta-detail-bonus-content .synopsis {
+  font-size: 10px;
+  @include media-breakpoint-up(lg) {
+    font-size: 12px;
+  }
+}
```
Comment (smitsanghvi21) [inline site/styles/_meta-detail.scss:582]: Also added this one, based on the changes in the Falcon repo.

File: site/styles/_meta-detail.scss (hunk 3 above)
Comment (stajs) [reply site/styles/_meta-detail.scss:355]: Yeah i think this is the right change

File: site/styles/_meta-detail.scss (hunk 4 above)
Comment (stajs) [reply site/styles/_meta-detail.scss:582]: Let's ditch this as it could have been their specific styling.
